modules/Sources/Dependencies/AddressBookClient/AddressBookInterface.swift

```diff
@@ -17,8 +17,8 @@ extension DependencyValues {
 
 @DependencyClient
 public struct AddressBookClient {
-    public let allLocalContacts: () throws -> AddressBookContacts
-    public let syncContacts: (AddressBookContacts?) async throws -> AddressBookContacts
-    public let storeContact: (Contact) throws -> AddressBookContacts
-    public let deleteContact: (Contact) throws -> AddressBookContacts
+    public let allLocalContacts: () throws -> (contacts: AddressBookContacts, remoteStoreResult: RemoteStoreResult?)
```

Comment:
I think this can return `RemoteStoreResult.notAttempted` rather than `nil`, meaning that it does not need to be optional.
```suggestion
    public let allLocalContacts: () throws -> (contacts: AddressBookContacts, remoteStoreResult: RemoteStoreResult)
```

Reply:
Updated